docs/introduction.rst

```diff
@@ -29,11 +29,11 @@ receiving WebSocket frames) can't trigger this.
 How does ASGI work?
 -------------------
 
-ASGI is structured as a double-callable - the first callable takes a ``scope``,
-which contains details about the incoming request, and returns a second,
-coroutine callable. This second callable gets given ``send`` and ``receive``
-awaitables that allow the coroutine to monitor incoming events and send
-outgoing events.
+ASGI is structured as a double-callable---the first callable takes a
+``scope``, which contains details about the incoming request, and returns a
+second, coroutine callable. This second callable is given ``send`` and
+``receive`` awaitables that allow the coroutine to monitor incoming events and
+send outgoing events.
```

Comment:
I've just realised I need to revise this part of the intro for the new spec. I'll do that once this lands.